docs/appendix.rst

```diff
@@ -59,18 +59,14 @@ This documentation uses a few special terms to refer to Python types:
 
    WBEM connection definitions file
    connections file
-      A file maintained by pywbemcli and managed by the
-      :ref:`Connection command group`.
-      The connections file defines a list of named connections, each of which
-      is either the set of parameters for connecting to a real WBEM server, or
-      the set of files for creating a mock WBEM server.
-      The file name of the connections file is
-      ``pywbemcli_connection_definitions.json`` and it is created by pywbemcli
-      in the current directory.
-      Multiple connections files may exist in different directories.
-      This allows connection definitions to be predefined and pywbemcli
-      commands executed against them using only the connection name (via the
```

Comment:
If this text from master is used as a basis, as suggested further down, change "connection name" to "connection definition name", as agreed to in PR #295.

Reply:
DONE